src/ai/backend/manager/models/image.py

```diff
@@ -597,11 +630,8 @@ async def load_all(
     ) -> Sequence[Image]:
         async with ctx.db.begin_readonly_session() as session:
             rows = await ImageRow.list(session, load_aliases=True)
-        items: List[Image] = []
+        items = [gen async for gen in cls.bulk_load(ctx, rows)]
```

Comment:
`gen` is not a generator but it's a single yielded item.
How about renaming it to `item`?